hack/e2e-common.sh

```diff
@@ -134,7 +134,16 @@ function cluster_kind_load {
 # $1 cluster
 # $2 image
 function cluster_kind_load_image {
-    $KIND load docker-image "$2" --name "$1"
+    # Check if the command to get worker nodes could succeeded
+    if ! $KIND get nodes --name "$1" > /dev/null 2>&1; then
+        echo "Failed to retrieve nodes for cluster '$1'."
+        return 1
+    fi
+    # filter out 'control-plane' node, usable only in multi-node single-cluster tests
```

Comment:
```suggestion
    # filter out 'control-plane' node, usable only in multi-node single-cluster tests
```
can you rephrase? IIUC now all CI tests would only use worker nodes?

Reply:
Yes, all clusters have also worker nodes.